app/views/govuk_publishing_components/components/_breadcrumbs.html.erb

```diff
@@ -1,17 +1,16 @@
 <%
   breadcrumbs ||= []
-  collapse_on_mobile ||= false
+  collapse_on_mobile ||= true
   inverse ||= false
-  collapse_class =  collapse_on_mobile && breadcrumbs.any? { |crumb| crumb[:is_page_parent] } ? "gem-c-breadcrumbs--collapse-on-mobile" : ""
+  collapse_class =  collapse_on_mobile ? "govuk-breadcrumbs--collapse-on-mobile" : ""
   invert_class = inverse ? "gem-c-breadcrumbs--inverse" : ""
   breadcrumb_presenter = GovukPublishingComponents::Presenters::Breadcrumbs.new(breadcrumbs, request.path)
 %>
-
 <script type="application/ld+json">
   <%= raw breadcrumb_presenter.structured_data.to_json %>
 </script>
 
-<div class="gem-c-breadcrumbs govuk-breadcrumbs <%= invert_class %>" data-module="track-click">
+<div class="gem-c-breadcrumbs govuk-breadcrumbs <%= invert_class %> <%= collapse_class %>" data-module="track-click">
```

Comment:
This is getting a little untidy, here's an example of an alternative way of doing this: https://github.com/alphagov/govuk_publishing_components/blob/master/app/views/govuk_publishing_components/components/_textarea.html.erb#L23-L25

Reply:
On further discussion the collapsing of the breadcrumb to first and last is the only behaviour. No option to change so true setting to become the only setting making things much simpler, will clean up the logic!